source/unified-test-format/tests/Makefile

```diff
@@ -23,6 +23,9 @@ load-balancers: HAS_AJV
 gridfs: HAS_AJV
 	@ajv test -s $(SCHEMA) -d "../../gridfs/tests/*.yml" --valid
 
+transactions: HAS_AJV
+	@ajv test -s $(SCHEMA) -d "../../transactions/tests/unified/*.yml" --valid
```

Comment:
I assume this was missed in https://github.com/mongodb/specifications/commit/4f9ca96588cf075a640d2332c0d5d3f59cb8df65 for [DRIVERS-1638](https://jira.mongodb.org/browse/DRIVERS-1638)?

Reply:
Yeah